### PR TITLE
Databases week2

### PR DESCRIPTION
**Pull request by Lokesh-sw**

### CODE REVIEW

File: databases/week2/homework_create_database.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    `quantity` int(10) NOT NULL
+    );
+
+CREATE TABLE `register` (
+    `book_id` int(10) unsigned,
+    `name` varchar(255) NOT NULL,
+    `writer` varchar(255) NOT NULL,
+    `date_published` int(10) NOT NULL,
+    `quantity` int(10) NOT NULL
+    );
```
Comment (Elia-Git) [inline databases/week2/homework_create_database.sql]: good here!

File: databases/week2/homework_additionalqueries.sql [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ 
+ -- Find how many tasks where created in each month, e.g. how many tasks were created in october, 
+ -- how many tasks were created in november, etc. (hint: use group by)
+ 
+SELECT
+  month(task.created) as month, COUNT(task.id) as task_total
+FROM
+  task
+GROUP BY month;
+
```
Comment (Elia-Git) [inline databases/week2/homework_additionalqueries.sql]: Small fix to differentiate months and tasks 
```
SELECT
  monthname(task.created) as month, COUNT(task.id) as task_total
FROM
  task
GROUP BY month;
```

File: databases/week2/homework_additionalqueries.sql [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+SELECT
+    task.title,
+    task.created,
+    user.name
+FROM
+    task
+JOIN
+    user
+WHERE
+    user.name = "Maryrose Meadows" AND month(created) = 9;
```
Comment (Elia-Git) [inline databases/week2/homework_additionalqueries.sql]: How are you joining tables task and user here..There should be something common (most of the time id) to join to different tables.

See below
```
SELECT user.name, task.title, task.created
FROM task
JOIN user_task ON task.id = user_task.task_id
JOIN user  ON user.id = user_task.user_id
WHERE user.name LIKE 'Maryrose Meadows' AND MONTH(created)=09;
```

File: databases/week2/homework_additionalqueries.sql [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+ -- Get all the tasks for 'Donald Duck' with status 'Not started'
+ 
+ SELECT 
+	task.title, task.status_id
+ FROM
+	task
+ JOIN 
+	user
+WHERE 
+	user.name LIKE "%Donald Duck%" AND task.status_id = 1;
```
Comment (Elia-Git) [inline databases/week2/homework_additionalqueries.sql]: Here also tables task and user are actually not joining.

File: databases/week2/homework_additionalqueries.sql [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+SELECT
+    task.title, user.email
+FROM
+    task
+JOIN
+    user
+WHERE
+    user.email LIKE "%@spotify.com";
+
```
Comment (Elia-Git) [inline databases/week2/homework_additionalqueries.sql]: Pls, read something about how the join tool works!
https://www.w3schools.com/sql/sql_join.asp

File: databases/week2/homework_excercise2.sql [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+ON students (name);
+
+
+-- Add a new column to the class table named status which can only have the following values: 
+-- not-started, ongoing, finished (hint: enumerations).
+
+ALTER TABLE class 
+ADD status 
+ENUM("not-started", "ongoing", "finished") NOT NULL
+;
```
Comment (Elia-Git) [inline databases/week2/homework_excercise2.sql:36]: Good here!

**Review by Elia-Git — CHANGES_REQUESTED** [review]
HI Lokesh,

Pls pay attention on your fromating your code as you are using extremely many(unwanted) lines and explore about JOIN .
Join tool: https://www.w3schools.com/sql/sql_join.asp
Code formatter: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

**Comment by Lokesh-sw** [comment]
> HI Lokesh,
> 
> Pls pay attention on your fromating your code as you are using extremely many(unwanted) lines and explore about JOIN . Join tool: https://www.w3schools.com/sql/sql_join.asp Code formatter: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

File: databases/week2/homework_excercise2.sql (hunk 6 above)
Comment (Lokesh-sw) [reply databases/week2/homework_excercise2.sql:36]: @Elia-Git  Thank you for your constructive feed back. I have made changes and learning more about join tool.

File: databases/week2/homework_additionalqueries.sql (hunk 3 above)
Comment (Lokesh-sw) [reply databases/week2/homework_additionalqueries.sql]: query solution you provided didnt work and trown error. could this be an alternative solution

```sql
SELECT task.title, task.created, user.name
FROM task 
JOIN user ON user.id = task.user_id
WHERE user.name LIKE 'Maryrose Meadows' AND MONTH(created)=09;
```

File: databases/week2/homework_additionalqueries.sql (hunk 4 above)
Comment (Lokesh-sw) [reply databases/week2/homework_additionalqueries.sql]: On this query could the following solution work?

```sql
SELECT 
	task.title, task.status_id, user.name, status.name
 FROM
	task
 JOIN 
	user ON user.id = task.user_id
JOIN 
status ON status.id = task.status_id
WHERE user.name LIKE "Donald Duck" AND status.name = 'Not started';
```

**Pull request merged**